public/form.js

```diff
@@ -3,25 +3,34 @@ const suButton = document.getElementById('submit');
 const checkbox = document.getElementById('accept');
 
 function formValidation() {
+
   if (validateEmail(email)) {}
   return false;
 
   if(activeSubmit(suButton)) {}
     return false;
-
 }
 
-//TODO: rewrite validateEmail without using regular expressions
-function validateEmail(email) {
-  const mailFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+function isValid(email) {
```

Comment:
With current implementation 
_gary.barmak@_ 
is valid email address

Let's fix that!